helper.php

```diff
@@ -163,6 +163,13 @@ function getFlags($setflags) {
             $flags['newentrytitle'] = $this->getLang('newentry');
         }
 
+        // Disable Pager
+        if(in_array('nopager', $setflags)) {
+            $flags['pager'] = False;
+        } else {
+            $flags['pager'] = True;
+        }
```

Comment:
One more minor thing I noticed: It would be nicer to have this as `$flags['pager'] = !in_array('nopager', $setflags);` - or is there any reason to have this if/else statement?